odoo/addons/base/views/res_users_views.xml

```diff
@@ -78,12 +97,16 @@
                         <field name="api_key_duration" groups="base.group_no_one"/>
                     </group>
                     <notebook>
-                        <page string="Users" name="users">
-                            <field name="users" context="{'search_default_filter_no_share':1}"/>
+                        <page string="Users" name="user_ids">
+                            <field name="user_ids" context="{'search_default_filter_no_share':1}"/>
                         </page>
                         <page string="Inherited" name="inherit_groups">
                             <label for="implied_ids" string="Users added to this group are automatically added in the following groups."/>
```

Comment:
Since it is a technical view, I find it easier to keep the label as-is. You could add the string besides/under it with a more detailed explication.

Reply:
I move the comment to the help field.